extensions/lifecycle/tasks/LifecycleTask.js

```diff
@@ -861,6 +861,44 @@ class LifecycleTask extends BackbeatTask {
      * @return {undefined}
      */
     _applyTransitionRule(params, log) {
+        // ZENKO-4173
+        // IF params.site is cold location:
+        // PUSH a Kafka entry to "cold-archive-req-<params.site>" topic
+        // Entry could look like:
+        // NOTE: the content of the entry should be review by the topic consumer's developer.
+        // [{
```

Comment:
@RachedBenMustapha I came up with this format but we might already describe what `cold-archive-req-<params.site>` topic's consumer is expecting.